api.planx.uk/saveAndReturn/sendEmail.test.ts

```diff
@@ -208,8 +208,7 @@ describe("Send Email endpoint", () => {
         .expect(200);
 
       const softDeleteSessionMock = queryMock.getCalls().find(mock => mock.id === "SoftDeleteLowcalSession");
-      expect(softDeleteSessionMock.response.errors).toBeUndefined();
```

Comment:
This wasn't a valid test - `softDeleteSessionMock.response.errors` does not exist.